docs/README.md

```diff
@@ -22,65 +22,103 @@ cd cluster-api-provider-nested
 kind create cluster --name=capn
 ```
 
-### Install `cert-manager`
+### Clone CAPI and build `clusterctl` tool
 
-Cert Manager is a soft dependency for the Cluster API components to enable mutating 
-and validating webhooks to be auto deployed. For more detailed instructions 
-go [Cert Manager Installion](https://cert-manager.io/docs/installation/kubernetes/#installing-with-regular-manifests).
+As a cluster API~(CAPI) provider, CAPN need build `clusterctl` tool
 
 ```shell
-kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.3.1/cert-manager.yaml
+git clone git@github.com:kubernetes-sigs/cluster-api.git
+cd cluster-api
+make clusterctl
```

Comment:
Can you wrap this in 3 backticks to make it a code block?

Reply:
I already make it in ``` :)